docs/migration-guides/6.0.md

```diff
@@ -3,12 +3,12 @@
 ## General
 
 1. Spectral has become a monorepo now and CLI <-> Core have been split into separate packages. If you intend to use a CLI version of Spectral make sure to install `@stoplight/spectral-cli`. Otherwise, go with `@stoplight/spectral-core` which is the JS API.
-2. No default ruleset is loaded by default as of now, thus if you don't supply a ruleset, Spectral will refuse to lint the document. If you use CLI, make sure to have a valid ruleset in your project or use a `--ruleset` flag. JS API has a method called `setRuleset` which can be used to set the relevant ruleset.
-3. Spectral is stricter overall and certain soft errors that previously were marked as warnings will throw now.
+2. No default ruleset is loaded by default as of now, thus if you don't supply a ruleset, Spectral refuses to lint the document. If you use CLI, make sure to have a valid ruleset in your project or use a `--ruleset` flag. JS API has a method called `setRuleset` which can be used to set the relevant ruleset.
```

Comment:
"No default ruleset is loaded by default as of now, thus if you don't supply a ruleset, Spectral refuses to lint the document."

Suggestion:

No ruleset is loaded by default. If you don't supply a ruleset, Spectral refuses to lint the document.